src/main/java/io/stargate/sgv2/jsonapi/service/projection/DocumentProjector.java

```diff
@@ -191,36 +228,32 @@ static PathCollector collectPaths(JsonNode def, boolean includeSimilarity) {
     }
 
     public DocumentProjector buildProjector() {
-      if (isDefaultProjection()) {
```

Comment:
Caller will use the short-cut, no need to check whether explicit definition happens to work like default projection.